tests/unit/test_pool.py

```diff
@@ -21,20 +22,37 @@ def make_filled_request_pool():
 def test_requests_pool_filters_relevant():
     pool = make_filled_request_pool()
     active_requests = pool.current_activities_requests
-    assert len(active_requests) == 7
+    assert len(active_requests) == 8
 
 
 def test_pairing_activities():
     pool = make_filled_request_pool()
     for activity in pool.activity_list:
         pool.make_pairs(activity)
+    assert len(pool.pairs['picnic']) == 2
+    assert len(pool.pairs['coffee']) == 2
+
+
+def test_ignored_pairs():
+    pool = make_filled_request_pool()
+    pool.users.ignore('christopher', 'olivia')
+    pool.users.ignore('christopher', 'martin')
+    pool.users.ignore('christopher', 'andrew')
+    pool.users.ignore('olivia', 'andrew')
+    pool.users.ignore('olivia', 'martin')
+    for activity in pool.activity_list:
```

Comment:
I will try to get the coverage report working over the weekend. That will help a lot to write tests.